addon/components/project-form/label.hbs

```diff
@@ -0,0 +1,9 @@
+<div class={{unless @noMargin "uk-margin"}}>
+  <label for={{@attr}} class={{if @required (configured-classes "text-bold")}}>
+    {{t (concat "ember-gwr.projectForm." @attr)}} {{if
+      @required
+      (t "ember-gwr.projectForm.required")
+    }}
+  </label>
+  {{yield}}
```

Comment:
If you ever wanted to have a horizontal layout, you'd need an element wrapping around the yield.

* https://getuikit.com/docs/form#layout
* https://getbootstrap.com/docs/4.0/components/forms/#horizontal-form

I recommend adding that wrapper now. Even if horizontal forms are not currently planned.